src/compiler/python_generator.cc

```diff
@@ -230,8 +230,10 @@ static void PrintAllComments(const DescriptorType* desc, Printer* printer) {
 
 bool PrintBetaServicer(const ServiceDescriptor* service, Printer* out) {
   out->Print("\n\n");
-  out->Print("class Beta$Service$Servicer(object):\n", "Service",
-             service->name());
+  out->Print("class Beta$Service$Servicer(object):\n"
+    "\"\"\"The Beta API is deprecated post the 14.0.x branch. It is "
```

Comment:
Let's say "is deprecated for gRPC Python 0.15.0 and later". Can we also say something like "This [class, function] was generated only to ease transition from grpcio<0.15.0 to grpcio>=0.15.0"? Can we put in newlines so the generated text doesn't run off the (generated) page?